__tests__/components/NavLink.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import NavLink from '../../components/NavLink'
+
+describe('NavLink Component', () => {
+  test('Should crash if given internal && blank props', () => {
+    // expect(() => render(<NavLink internal blank />)).toThrow()
```

Comment:
why do we need an empty test?

Reply:
In my comments I mentioned how I wasn't sure how to test this properly lol, I will either update it when I figure it out or remove it completely if we think it's not necessary

Reply:
storybook comment. Just render the different props is good enough.

Reply:
removed test